src/parser/parse_evaluate.c

```diff
@@ -962,6 +962,7 @@ pt_evaluate_tree_internal (PARSER_CONTEXT * parser, PT_NODE * tree, DB_VALUE * d
       if (val)
 	{
 	  (void) db_value_clone (val, db_values);
+	  db_values->need_clear = true;
```

Comment:
Why do you need it?

Reply:
Because otherwise memory will be leaked (eval_value from do_evaluate_insert_values). pt_evaluate_tree_having_serial (execute_statement.c:16549) allocates memory for eval_value and never releases it.

Reply:
I expect db_value_clone sets the flag.
Do I miss something?

Reply:
ah, yes, my bad, it properly sets it